### ISSUE TITLE
Synth data training problem

### ISSUE THREAD

**Issue by zhenx** [backtime92/CRAFT-Reimplementation#24]
Hi, I tried your code trainSyndata.py on data http://www.robots.ox.ac.uk/~vgg/data/scenetext/. However, after 16 epochs, the hmean on IC2013 still around 65%, which is far below your results(76.33%). Could you suggest what the possible problems are, please?

**Comment by backtime92** [comment]
@zhenx I used the old gaussian map method to train syndata, I found the new gaussian map method pre-train model is not better than old's. So I think you can just use my pre-train model to train MLT or IC15 data. And if you need, I can share you the old gaussian map method.

**Comment by ddm-456** [comment]
@backtime92 I'm confused. So if I want to get the 82% Hmean result on ICDAR15 from scratch.  I should first train the pretrained model on SynthText data with the old gaussian method (which is the same as the model you provided: [link](https://pan.baidu.com/s/1MaznjE79JNS9Ld48ZtRefg)), and then finetune it on ICDAR15 using the new gaussian method?

**Comment by backtime92** [comment]
@ddm-456 yes,and if you use the scratch from the new gaussian method, it can be ~81%. But i think you can just use the syndata pre-trained model which i had given the link.

**Comment by zhenx** [comment]
@backtime92 Thanks for your kind reply. Yes, could you please share your old version of  the heatmap code? I am wondering what the main different between these two versions are. Could you elaborate a little bit? One more question is about your note in the readme "Note:We have solved the problem about detecting big word. Now we are training the model. And any issues or advice are welcome.". How did you deal with that problem of missing big words? What's the main action you took exactly? Thanks.

**Comment by backtime92** [comment]
@zhenx I need to find it. there are so many versions code, I will check it and upload it.

**Comment by quangphammessi** [comment]
Hi @backtime92 thank you for opening your reimplementation code of CRAFT. I want to retrain CRAFT model on Synthtext dataset but I cannot achieve the score you got. Could you share to me the gaussian method that you used for your training? Thanks.

**Comment by backtime92** [comment]
@quangphammessi Sorry for you problem, the Syndata model I trained by old version code, the new is not suitable for training the syndata model, I think you can train the IC15 and MLT data model by using my Syndata pre-trained model. You can see my readme.md note.

**Comment by quangphammessi** [comment]
@backtime92 Ok. I see. Thank you.

**Comment by huntu0042** [comment]
hi @backtime92 . thank you for code and advice. I wonder that how many epoch did you train to make Syndata Model. I want to check training time to train my custom model.  thank you.

**Comment by nightfuryyy** [comment]
> @zhenx I need to find it. there are so many versions code, I will check it and upload it.

hi @backtime92 . do you upload it ?
